status/format.go

```diff
@@ -29,17 +30,20 @@ const commentTemplate = `### {{ .Title }}
 
 {{ .Summary }}`
 
-const summaryTemplate = `| Task | Duration | Test Suite | Status | Details |
-| --- | --- | --- | --- | --- |
-{{- range $tr := .TaskRuns }}
-| {{ formatTaskName $tr }} | {{ $tr.GetDuration.String }} | {{ formatNamespace $tr }} | {{ formatStatus $tr }} | {{ formatDetails $tr }} |
+const summaryTemplate = `| Task | Duration | Test Suite | Status | Details | PipelineURL |
```

Comment:
Do we need to have the long pipelinerun url in each row?

Reply:
Hmm, I think you are right, maybe have it separated from the table somewhere.

Reply:
Updated

Reply:
Thanks. Or as suggested in https://github.com/redhat-appstudio/integration-service/pull/609#discussion_r1517691389 and [STONEINTG-757](https://issues.redhat.com/browse/STONEINTG-757)?

Reply:
Ah, that is helpful, thanks Hongwei!